src/common/components/nav/nav.component.js

```diff
@@ -39,10 +39,22 @@ class NavController{
     this.sessionModalService = sessionModalService;
 
     this.imgDomain = envService.read('imgDomain');
-    this.navFeed = '/bin/cru/site-nav.json';
   }
 
   $onInit() {
+    //init attributes
```

Comment:
Huh that's interesting. The constructor should run and then once the bindings are ready, $onInit should run. You'd think a non-existent binding wouldn't override the value. I'm not sure... The type of binding is the same as the loading component... I suppose you could put a breakpoint at the end of the constructor and the beginning of $onInit to make sure the values are as you expect then to be but I imagine it gets set to undefined between there.